apps/src/gamelab/GameLab.js

```diff
@@ -171,14 +183,18 @@ GameLab.prototype.init = function (config) {
     });
   }.bind(this);
 
-  ReactDOM.render(React.createElement(AppView, {
+  this.reduxStore_.dispatch(actions.setInitialLevelProps({
     assetUrl: this.studioApp_.assetUrl,
     isEmbedView: !!config.embed,
-    isShareView: !!config.share,
-    renderCodeWorkspace: renderCodeWorkspace,
-    renderVisualizationColumn: renderVisualizationColumn,
-    onMount: onMount
-  }), document.getElementById(config.containerId));
+    isShareView: !!config.share
+  }));
+
+  ReactDOM.render(<Provider store={this.reduxStore_}>
+    <GameLabView
+      renderCodeWorkspace={renderCodeWorkspace}
+      renderVisualizationColumn={renderVisualizationColumn}
+      onMount={onMount} />
```

Comment:
Note: If we get to the point that renderCodeWorkspace/renderVisualization column return React components, we should provide those as children instead of props, i.e.
```
<GameLabView onMount={onMount}>
  {renderCodeWorkspace()}
  {renderVisualizationColumn}
</GameLab>
```
I don't think we're there yet though. I wonder if we should use a different word than render to make it clear that this is a non-React rendering?